src/chat/Client/User.java

```diff
@@ -1,9 +1,9 @@
-package chat.Client;
+package src.chat.Client;
```

Comment:
Настрой IDE, некорректный импорт. Источник кода должен быть src/, а не корень.
```suggestion
package chat.Client;
```